### PR TITLE
Added email settings; Added global email module. closes #17

### LINKED ISSUE #17: Add gmail SMTP api to send emails

### PR DESCRIPTION
**Pull request by sk364**
[Technical Implementation Details](https://github.com/Velle-log/hs-api/wiki/%5BTechnical%5D-%5BImplementation%5D-Email)

### CODE REVIEW

**Comment by Samy-33** [comment]
@sk364, this implements async email sending?  
If that's the case, can we put #16 into backlog? We can take that up in future sprints.

**Review by Samy-33 — DISMISSED** [review]

File: globals/utils/email.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+from django.core.mail import send_mail
+
+
+# TODO: make this function asynchronous
+# TODO: add logging on success/error of the sending email
+def send_async_mail(subject, message, recipients):
```
Comment (Samy-33) [inline globals/utils/email.py]: Please rename the function just `send_mail`  
We will use this method in `tasks.py`, which is celeries standard way to define async tasks.

**Review by Samy-33 — CHANGES_REQUESTED** [review]

File: globals/utils/email.py (hunk 1 above)
Comment (Samy-33) [reply globals/utils/email.py]: Sorry not required.

**Review by Samy-33 — APPROVED** [review]

File: globals/utils/email.py (hunk 1 above)
Comment (sk364) [reply globals/utils/email.py]: Done!

**Pull request merged**